frontend/app/LetterForm.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import React, { useState, ChangeEvent, FormEvent } from "react";
+import { solveBestTexts, SolveRequestPayload, SolveResponse } from "./api";
+
+// Generate the alphabet letters
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("");
+
+const AlphabetForm: React.FC = () => {
+  // Initialize state with letter counts set to 0
+  const initialFormValues: SolveRequestPayload = {
```

Comment:
The type of `initialFormValues` and `formValues` could maybe be just `Record<string, number>` as the form only contains letters and counts, not other attributes of `SolveRequestPayload`.

Reply:
Good point, this was adding unnecessary complexity to the structure and its handling elsewhere in the component. Changed it to `Record<string, number>` as suggested and it is wrapped to a `SolveRequestPayload` type only when the API call for the solver is made.